spec/features/users/user_show_spec.rb

```diff
@@ -1,20 +1,17 @@
 include Warden::Test::Helpers
 Warden.test_mode!
 
-# Feature: User profile page
-#   As a user
-#   I want to visit my user profile page
-#   So I can see my personal account data
-feature 'User profile page', :devise do
-
+feature 'User profile page', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.